doc/user/zebra.rst

```diff
@@ -58,6 +58,26 @@ Besides the common invocation options (:ref:`common-invocation-options`), the
 
 .. _interface-commands:
 
+Configuration Addresses behaviour
+=================================
+
+At startup, when running *Zebra*, the daemon will first discover the
+underlying networking objects from the operating system. This includes
+interfaces, addresses of interfaces, static routes. Then, it will exec
```

Comment:
If these aren't the only things it discovers please add `etc.` after `static routes`.  If they are, please put `and` before `static routes`.

Please replace `exec` with `read`.

Reply:
ok